manifests/u/UniversityOfEdinburgh/QuPath/0.2.3/UniversityOfEdinburgh.QuPath.locale.en-US.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: UniversityOfEdinburgh.QuPath
+PackageVersion: 0.2.3
+PackageLocale: en-US
+Publisher: Unknown
+PublisherUrl: https://www.ed.ac.uk/
+PackageName: QuPath-0.2.3

```

This is technically correct, but it dosn't look correct in `winget list` or `winget upgrade`
The ProductCode will make it match for now tho
```suggestion
PackageName: QuPath
```

---

What about `Publisher: Unknown`? Does the publisher need to match or can we also put a proper value there?

---

It does need to match, Unknown is better than a blank field